io.github.babaissarkar.ssplot.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "io.github.babaissarkar.ssplot",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": "org.freedesktop.Sdk.Extension.openjdk17",
+    "command": "ssplot",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

Comment:
does this really support wayland? experience show that Java doesn't.

Reply:
Okay, will remove that.